spec/factories/users.rb

```diff
@@ -0,0 +1,9 @@
+FactoryBot.define do
+  factory :user do
+    nickname { Faker::Name.name }
+    email { 'foo@bar.com' }
```

Comment:
I think you can also use faker to generate email for you. That is if you want to.